channels/reducer.js

```diff
@@ -41,31 +42,38 @@ function createReducer (config) {
       }
     }),
     [unsetChannels]: (state, action) => loop(state, Effects.batch(
-      map(action.payload, channel => Effects.constant(unsetChannel(channel))))),
+      map(action.payload, id => Effects.constant(unsetChannel(id))))),
     [unsetChannel]: (state, action) => {
-      const channel = action.payload
-      const { id, channels, clips } = channel
+      const channel = state.records[action.payload]
+      const { id, channelIds = [], clipIds = [] } = channel
 
-      const nextRecords = state.records
-      delete nextRecords[id]
+      // remove this channel from all parent channels
+      // TODO: is this necessary? what if we just assume an empty id doesnt exist?
```

Comment:
is it possible to handle this in the clip's reducer? looking at how [`reduxLoop/combineReducers`](https://github.com/redux-loop/redux-loop/blob/v2.2.2/modules/combineReducers.js) works, it seems like you could handle the action and return a new model for that state there.